packages/components/src/components/wallet-card/wallet-card.tsx

```diff
@@ -0,0 +1,113 @@
+                    [`wallet-card__container-fade--${state}`]: state,
+                })}
+            />
+            {size === 'small' && <IconComponent size={size} icon_type={wallet.icon_type} icon={wallet.icon} />}
+            {size !== 'small' && (
+                <div className={`wallet-card__content wallet-card__content--${size}`}>
+                    <div className='wallet-card__top-wrapper'>
+                        <IconComponent size={size} icon_type={wallet.icon_type} icon={wallet.icon} />
+                        {wallet.jurisdiction_title === 'virtual' ? (
+                            <Badge label={localize('Demo')} type='contained' background_color='blue' />
+                        ) : (
+                            <Badge
+                                custom_color='var(--text-prominent)'
+                                label={wallet.jurisdiction_title.toUpperCase()}
+                                type='bordered'
+                            />
+                        )}
+                    </div>
+
+                    <div className='wallet-card__bottom-wrapper'>
+                        {state !== 'add' && state !== 'added' ? (
+                            <React.Fragment>
+                                <Text color='prominent' size={isMobile() ? 'xxxxs' : 'xxxs'}>
+                                    {wallet.name}
+                                </Text>
+                                <Text color='prominent' weight='bold' size={isMobile() ? 'xxs' : 'xs'}>
+                                    {wallet.balance} {wallet.currency}
+                                </Text>
+                            </React.Fragment>
+                        ) : (
+                            <Button
+                                className={`wallet-card__wallet-button wallet-card__wallet-button--${state}`}
+                                classNameSpan='wallet-card__wallet-button-text'
+                                icon={
+                                    <Icon
+                                        className='wallet-card__wallet-button-icon'
+                                        custom_color='$color-black-1'
+                                        icon={state === 'added' ? 'IcCheckmarkBold' : 'IcAddBold'}
+                                        size={12}
+                                    />
+                                }
+                                text={state === 'added' ? localize('Added') : localize('Add')}
+                                is_disabled={state === 'added'}
+                            />
+                        )}
+                    </div>
+                </div>
+            )}
+        </div>
+        {state === 'active' && (
+            <div className={`wallet-card__active-icon wallet-card__active-icon--${size}`}>
+                <Icon
+                    color='brand'
+                    data_testid='ic-checkmark-circle'
```

Comment:
Question: Shouldn't data_testid be created using snake_case and the dt prefix?
```suggestion
                    data_testid='dt_ic_checkmark_circle'
```